packages/AttributeAwarePhpDoc/src/AttributeAwareNodeFactory/AttributeAwareGenericTagValueNodeFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Rector\AttributeAwarePhpDoc\AttributeAwareNodeFactory;
+
+class AttributeAwareGenericTagValueNodeFactory implements \Rector\AttributeAwarePhpDoc\Contract\AttributeNodeAwareFactory\AttributeNodeAwareFactoryInterface
+{
+    public function getOriginalNodeClass(): string
+    {
+        return \PHPStan\PhpDocParser\Ast\PhpDoc\GenericTagValueNode::class;
+    }
+    public function isMatch(\PHPStan\PhpDocParser\Ast\Node $node): bool
+    {
+        return is_a($node, \PHPStan\PhpDocParser\Ast\PhpDoc\GenericTagValueNode::class, true);
+    }
+    public function create(\PHPStan\PhpDocParser\Ast\Node $node): \Rector\BetterPhpDocParser\Contract\PhpDocNode\AttributeAwareNodeInterface
+    {
+        return new \Rector\AttributeAwarePhpDoc\Ast\PhpDoc\AttributeAwareGenericTagValueNode();
+    }
+}
```

Comment:
This file doesn't seem to match coding standards. Is it on purpose ?

Reply:
I didn't run coding standard in this commit.
It fixes it correctly, I tried it